src/textual/dom.py

```diff
@@ -71,6 +78,19 @@ def parent(self) -> DOMNode:
         assert isinstance(self._parent, DOMNode)
         return self._parent
 
+    @property
+    def screen(self) -> "Screen":
+        """Get the current screen."""
```

Comment:
```suggestion
        """Get the screen that this node is contained within. Note that this may not be the currently active screen within the app."""
```